plugin/externalPluginRoot.go

```diff
@@ -42,18 +48,22 @@ func (r *externalPluginRoot) Init() error {
 			"the plugin root",
 			err,
 			stdout,
-			"{\"name\":\"plugin_name\",\"methods\":[\"list\"]}",
+			"{}",
 		)
 	}
+
+	// Fill in required fields with data we already know.
+	decodedRoot.Name = r.Name()
+	decodedRoot.Methods = []string{"list"}
```

Comment:
This would need to be changed to not overwrite Methods if it was supplied when we implement the data-driven hierarchy described in https://github.com/puppetlabs/wash/projects/9.

Reply:
Bit worried that we're assuming too much too soon re: list being the only operation roots can support. Think we should keep `init` the way it is for now, and maybe add some validation to make sure that the returned name matches what we got?

Reply:
I was erring away from handling backwards-compatibility, but I guess in this case it's simple enough to warn about potential issues.

Reply:
Let's do that then. Would rather keep `init`'s output symmetric with a single entry JSON in `list`'s output.